pkg/webhook/work/mutating.go

```diff
@@ -67,6 +67,11 @@ func (a *MutatingAdmission) Handle(_ context.Context, req admission.Request) adm
 			return admission.Errored(http.StatusInternalServerError, err)
 		}
 
+		// We only set these labels and annotations on resources propagated down.
```

Comment:
We don't usually say `propagate down` or `propagate up`, so proposing the comments like:
```
// Skip label/annotate the workload of Work that is not intended to be propagated.
```